modules/manage/pages/cluster-maintenance/cluster-diagnostics.adoc

```diff
@@ -11,7 +11,24 @@ When anomalous behavior arises in a cluster and you're trying to figure out whet
 
 Self-test runs a set of benchmarks to determine the maximum performance of a machine's disks and network connections. For disks, it runs throughput and latency tests by performing concurrent sequential operations. For networks, it selects unique pairs of Redpanda nodes as client/server pairs, then it runs throughput tests between them. Self-test runs each benchmark for a configurable duration, and it returns IOPS, throughput, and latency metrics.
 
-=== Self-test command examples
+== Cloud storage tests
+
+If you use xref:manage:tiered-storage.adoc[Tiered Storage], run self-test to verify that you have configured your cloud storage accounts correctly.
+
+Self-test performs the following tests to validate cloud storage configuration:
+
+. Upload an object (a random buffer of 1024 bytes) to the cloud storage bucket.
+. List objects in the bucket.
+. Download the uploaded object from the bucket.
+. Download the uploaded object's metadata from the bucket.
+. Delete the uploaded object from the bucket.
+. Upload and then delete multiple objects (random buffers) at once from the bucket. 
+
+See the xref:reference:rpk/rpk-cluster/rpk-cluster-self-test.adoc[`rpk cluster self-test`] reference for cloud storage test details.
```

Comment:
Link to test start instead.

Reply:
@WillemKauf @Lazin 
 surprised we don't test the throttling behavior based on bucket prefixes.  Is that something we can easily test?

Reply:
@mattschumpert 

Currently the `cloud storage self-test` is more intended for configuration verification, rather than being a true benchmark/stress test.

We can always expand upon this in the future to cover more rigorous workloads.